### PR TITLE
Update JS files and CSS files

### PR DESCRIPTION
**Pull request by KDuverge**
What it do baybeeee

- Broke out JS files
- Updated CSS and made it mobile responsive

Let me know what you think!

### CODE REVIEW

**Comment by KDuverge** [comment]
- Just tested on my phone, and everything looks good. I made the `recently-played-container` scrollable.

File: src/js/app.js [hunk 1]
```diff
@@ -1,25 +1,12 @@
+import './device.js';
+import { setTheme } from './themes.js';
+import { nowPlayingHtml, recentlyPlayedHtml, displayNotPlayingHtml } from './utils.js';
+
```
Comment (dontfollowsean) [inline src/js/app.js]: Can't use these import statements because my gulp config. Making some adjustments

File: src/js/utils.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    return `
+        <div class="now-playing-container">
+        <label>Now Playing</label>
+        <div class="now-playing-info">
+            <div class="now-playing-title" onclick="window.open('${song.url}', '_blank')">${song.title}</div>
+            <div class="now-playing-artist">${getArtistText(song.artist)}</div>
+        </div>
+        </div>
+        <div class="now-playing-img" onclick="window.open('${song.url}', '_blank')">
+        <img src=${image.url} alt=${song.title} />
```
Comment (dontfollowsean) [inline src/js/utils.js]: Where is `image` defined?

**Comment by dontfollowsean** [comment]
Overall this looks great but I'm going to make a few tweaks when I get a sec.

**Pull request merged**